doc/tor.1.txt

```diff
@@ -1020,6 +1020,21 @@ The following options are useful only for clients (that is, if
     The .exit address notation, if enabled via MapAddress, overrides
     this option.
 
+[[MiddleNodes]] **MiddleNodes** __node__,__node__,__...__::
```

Comment:
If this is going to get merged, we should mark it as experimental-only for some reason, and tell people what they should use instead for non-experimental use.  We may also want to say explicitly that it might go away in the future.